Lib/pathlib.py

```diff
@@ -114,7 +114,8 @@ class _WindowsFlavour(_Flavour):
 
     is_supported = (os.name == 'nt')
 
-    drive_letters = set('abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ')
+    drive_letters = set('abcdefghijklmnopqrstuvwxyz'
+                        'ABCDEFGHIJKLMNOPQRSTUVWXYZ')
```

Comment:
This change is unnecessary and not an improvement.  Please leave as-is.
Adding a multi-line statement with implicit string concatenation doesn't make the code simpler.

Reply:
This line is too long and don't conform PEP 8.